README.md

```diff
@@ -19,6 +20,7 @@ This library contains several components in different directories:
 - table_processing: Table extraction tools
 - tests: pytest test cases
 - src: contains deprecated software from an earlier version of this library. This directory is planned for removal as it is being replaced with file_processing tools
+- table_trials_results: Results in .xlsx format of the tests run on generated tables with a variety of randomized parameters including rows, columns, vertical/horizontal lines, font size, row height, margin, orientation, and special characters. Parameter values and performance metrics are tracked for each of the roughly 7000 generated tables along with overall summaries for each individual metric (mean, stdev, min, max, median, etc.).
```

Comment:
Add a quick explanation of how to run the test, a summary of the results, and where the complete resultls are.